iothub/service/src/Registry/ModulesClient.cs

```diff
@@ -21,8 +21,6 @@ public class ModulesClient
         private readonly HttpClient _httpClient;
         private readonly HttpRequestMessageFactory _httpRequestMessageFactory;
 
-        private const string ModulesRequestUriFormat = "/devices/{0}/modules/{1}";
```

Comment:
I'd like to keep these strings as constants. If we have code like 

```csharp
return new Uri($"/devices/{deviceId}/modules/{moduleId}", UriKind.Relative);
```

It actually allocates a new string on the heap each time. By keeping it as a constant, we just reference the same point on the heap repeatedly. So what you should have below is something like

```csharp
string.Format(CultureInfo.InvariantCulture, ModulesRequestUriFormat, deviceId, moduleId);
```

Reply:
added the changes